nkpremices/EPIC-Mail
# #164427868 User should check the "remember me " on login also should make a search on the dashboard

#### What does this PR do?
Have a remember me on the signUpIn page and a search input on the dashboard

#### Description of Task to be completed?
Have the remember me and the search input working

#### How should this be manually tested?
After cloning the repo; open the index.html file to see the remember me; or cd to the html folder then open the dashboard.html page 

#### What are the relevant pivotal tracker stories?
#164241288
#164297786
#164242545
